cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapperWrapper.java

```diff
@@ -159,7 +159,7 @@ public boolean nextKeyValue() throws IOException, InterruptedException {
         boolean result = super.nextKeyValue();
         if (++processedRecords > flushFreq) {
           try {
-            LOG.info("Flushing dataset operations...");
+            LOG.trace("Flushing dataset operations...");
```

Comment:
Well, since it's +1 every time, probably doesn't matter... also, it's a trace, doesn't matter too much.